libs/email-service/src/main/java/uk/gov/di/ipv/core/library/service/EmailService.java

```diff
@@ -38,6 +38,15 @@ public EmailService(ConfigService configService, NotificationClient notification
 
     public void sendUserTriggeredIdentityResetConfirmation(
             String userEmailAddress, String fullName) {
+        sendUserTriggeredIdentityResetConfirmation(
```

Comment:
I wouldn't combine these like this. At the moment they have the same personalisation, but there's no guarantee things will stay that way. I'd copy `sendUserTriggeredIdentityResetConfirmation` and just change the template parameter.